src/test/scala/za/co/absa/spark/commons/implicits/StructFieldImplicitsTest.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spark.commons.implicits
+
+import org.apache.spark.sql.types.{Metadata, StringType, StructField}
+import org.scalatest.funsuite.AnyFunSuite
+import za.co.absa.spark.commons.implicits.StructFieldImplicits.StructFieldMetadataEnhancement
+
+class StructFieldImplicitsTest extends AnyFunSuite {
+
+  def fieldWith(value123: String) = {
+    val value1 = s"""{ \"a\" : ${value123} }"""
```

Comment:
```suggestion
    val value1 = s"""{ "a" : ${value123} }"""
```
I think escaping in of `"`s inside of `"""`s is unnecessary.